### PR TITLE
[Apps] updating imports

### PR DESCRIPTION
**Pull request by philbucher**
after #6556

### CODE REVIEW

File: applications/SwimmingDEMApplication/SwimmingDEMApplication.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
 from KratosSwimmingDEMApplication import *
 
-from KratosMultiphysics import _ImportApplicationAsModule
+from KratosMultiphysics import _ImportApplication
 application = KratosSwimmingDEMApplication()
 application_name = "KratosSwimmingDEMApplication"
-application_folder = "SwimmingDEMApplication"
 
-_ImportApplicationAsModule(application, application_name, application_folder, __path__)
+_ImportApplication(application, application_name)
```
Comment (roigcarlo) [inline applications/SwimmingDEMApplication/SwimmingDEMApplication.py:8]: We should change so to be like the others...

File: applications/RANSApplication/RANSApplication.py [hunk 2]
```diff
@@ -8,9 +8,8 @@
     from KratosMultiphysics.TrilinosApplication import *
 from KratosRANSApplication import *
 
-from KratosMultiphysics import _ImportApplicationAsModule
+from KratosMultiphysics import _ImportApplication
```
Comment (roigcarlo) [inline applications/RANSApplication/RANSApplication.py:11]: That one as well

**Comment by roigcarlo** [comment]
Ok better to make a general issue. Seems that half of the apps are importing Kratos as KM and then calling the __import function and the other half are importing directly the function.

Since we are changing this, we should take the chance and set a common ground.

I have no particular preference for any method (but probably importing only the function is faster than importing the whole Kratos)

**Comment by philbucher** [comment]
> I have no particular preference for any method (but probably importing only the function is faster than importing the whole Kratos)

Ok then I will do that

**Comment by philbucher** [comment]
done @roigcarlo now it should be consistent

**Review by roigcarlo — DISMISSED** [review]
Ok to me now !

**Comment by philbucher** [comment]
@roigcarlo I missed two apps, can you reapprove pls?

**Review by roigcarlo — APPROVED** [review]

**Pull request merged**